src/driver/sqlite-abstract/AbstractSqliteQueryRunner.ts

```diff
@@ -1149,8 +1167,12 @@ export abstract class AbstractSqliteQueryRunner extends BaseQueryRunner implemen
             downQueries.push(this.createIndexSql(oldTable, index));
         });
 
+        const [databaseNew, tableNameNew] = this.splitTablePath(newTable.name)
+        // @ts-ignore // unused var
```

Comment:
Can `_` be used for unused vars?

Reply:
I checked now the syntax is actually to not put in any variable:

`[ , tableNameNew]`